src/main/java/com/btkelly/gnag/models/ViolationComment.java

```diff
@@ -25,7 +25,7 @@
 
     public ViolationComment(boolean failBuild, String commentMessage) {
```

Comment:
It is in the auto complete for IDEA but I get the following compilation error when building

/Users/bobbake4/development/gnag/src/main/java/com/btkelly/gnag/models/ViolationComment.java:18: error: package com.sun.javafx.beans.annotations does not exist
import com.sun.javafx.beans.annotations.NonNull;
                                       ^
/Users/bobbake4/development/gnag/src/main/java/com/btkelly/gnag/models/ViolationComment.java:28: error: cannot find symbol
    public ViolationComment(boolean failBuild, @NonNull String commentMessage) {
                                                ^
  symbol:   class NonNull
  location: class ViolationComment